community/php7-mustache/APKBUILD

```diff
@@ -0,0 +1,55 @@
+# Contributor: Renoir Boulanger <hello@renoirboulanger.com>
+# Maintainer: John Boehr <jbboehr@gmail.com>
+pkgname=php7-mustache
+pkgver=0.7.2
+pkgrel=0
+_phpver=${pkgname#php}
+_phpver=${_phpver%%-*}
+pkgdesc="PHP$_phpver extension: C++ implementation of Mustache"
+url="https://github.com/jbboehr/php-mustache"
+arch="all"
+license="MIT"
+# XXX: sudo apk add --no-cache --update --repository http://dl-cdn.alpinelinux.org/alpine/edge/community php7-dev php7-common
+depends="
```

Comment:
It shouldn't break builds, as far as I know, it's just more of a style thing if the lines do not need to be split. :)